### ISSUE TITLE
Необходим сервис для определения версии БСП

### ISSUE THREAD

**Issue by DoublesunRUS** [1C-Company/ssl-support#35]
Пытался сделать свои проверки по правильности внедрения БСП. См. https://github.com/marmyshev/edt-plugins/issues/15
В процессе реализации проверок столкнулся с необходимостью определять версию БСП, которая встроена в основную конфигурацию.

**Comment by serjevski** [comment]
так там в БСП функция есть которая возвращает версию... Или я не о том?

**Comment by marmyshev** [comment]
Видимо нужно в плагине определять версию БСП и в зависимости от этого - варианты поведения плагина...

**Comment by DoublesunRUS** [comment]
Да, нужно в плагине определять версию БСП. И делать проверки в зависимости от этой версии.

**Comment by VitaliyVS-2020** [comment]
Функция, которая возвращает версию БСП, лежит в известном модуле в известной функции. Там можно посмотреть?
